app/src/test/java/org/oppia/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -280,6 +287,34 @@ class StateFragmentLocalTest {
     }
   }
 
+  @Test
+  fun testStateFragment_submitTwoWrongAnswers_hintAvailable_prevState_hintNotAvailable() {
```

Comment:
Does this fail without the fix? Also, is this the right case? I thought we needed a case when a previous state can show the hint bulb, then we navigate back to it. It shouldn’t show the hint bulb again. Can you add a test case for that, too?

Reply:
The use case we have is:
1. The user submit wrong answers
2. Hint bulb visible
3. Going to prev state
4. Hint bulb should not visible
5. Back to the current state
6. Hint bulb should visible again

Reply:
Thanks--this does look correct.